### PR TITLE
Update kstest-runner container for Fedora 39 base image

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: containers/runner/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ RUN dnf -y update && \
     rsync \
     git \
     virt-install \
+    libvirt-daemon \
```
Comment (KKoukiou) [inline containers/runner/Dockerfile:11]: Not sure i follow but sure this is needed to run VMs.

Comment (rvykydal) [reply containers/runner/Dockerfile:11]: It contains libvirtd, in F38 it was pulled in by some other virt package I guess, in F39 I think there was some packages consolidation in libvirt.

**Review by jkonecny12 — APPROVED** [review]
This is great! Thanks a lot!

**Pull request merged**